Interoperability/python/qsharp/__init__.py

```diff
@@ -47,27 +49,24 @@
     SIMULATOR_COLOR = ca.Fore.BLUE + ca.Back.WHITE
     RESET_COLOR = ca.Style.RESET_ALL
 except:
-    ca = None
+    CA = None
     SIMULATOR_COLOR = ""
     RESET_COLOR = ""
 
 try:
     import qutip as qt
 except:
-    qt = None
+    QT = None
 
 try:
     from IPython.display import display
 except:
-    def display(value):
+    def display():
```

Comment:
Intention of this piece of code seems to be making the method silent in case the import fails. This change breaks the code if the import fails. Did you test this scenario ?

Reply:
I didn't, but i see why this wouldn't work. Procedures should not be defined with unused arguments. In my opinion, we should not catch anything if `from IPython.display import display` fails. Maybe put this in a class specifically for jupyter notebooks?

Reply:
I'm with @RolfHuisman, this is meant as a fallback to make sure that calls to `display` fail silently when run outside a Jupyter environment.